specs/olm.spec

```diff
@@ -20,6 +20,16 @@ Steps:
     | S.NO | resource_dir                   |
     |------|--------------------------------|
     | 1    | testdata/hub/tektonhub.yaml    |
+  * Create secrets for Tekton Results
+  * Switch to project "openshift-pipelines"
+  * Apply  
+    | S.NO | resource_dir                   |
+    |------|--------------------------------|
+    | 1    | testdata/pvc/tekton-logs.yaml  |
+  * Apply
+    | S.NO | resource_dir                   |
+    |------|--------------------------------|
+    | 1    | testdata/results/result.yaml   |
   * Configure GitHub token for git resolver in TektonConfig
   * Configure the bundles resolver
   * Validate pipelines deployment
```

Comment:
Installation gets stuck on `Validate RBAC` - I guess that it's caused by the fact that you switched the namespace and SA is not being created in namespace `openshift-pipelines` (expected behavior). You probably need to switch back or create a new namespace.

@VeereshAradhya we should refactor how we work with workspaces in release-tests, it's causing more and more trouble.

```
      * Verify TektonAddons Install status       ...[PASS]
2023/12/06 13:18:35 Verifying that TektonConfig status is "installed"
2023/12/06 13:18:41 Verifying that service account pipeline exists
2023/12/06 13:18:46 Verifying that service account pipeline exists
2023/12/06 13:18:51 Verifying that service account pipeline exists
2023/12/06 13:18:56 Verifying that service account pipeline exists
2023/12/06 13:19:01 Verifying that service account pipeline exists
2023/12/06 13:19:06 Verifying that service account pipeline exists
```